app/components/publications/publication/decisions/info/decisions-info-panel.hbs

```diff
@@ -81,7 +81,7 @@
                   {{else}}
                     <Auk::Datepicker
                       @date={{@publicationFlow.agendaItemTreatment.startDate}}
-                      @onChange={{this.setDecisionDate}}
+                      @onChange={{set this "agendaItemTreatment.startDate"}}
```

Comment:
Deze PR is niet de oorzaak, maar bij `@date` kunnen we dan ook best `this.agendaItemTreatment.startDate` gebruiken ipv `@publicationFlow.agendaItemTreatment.startDate`

Reply:
Bij set `{{set this.agendaItemTreatment "startDate}}`